Doc/deprecations/pending-removal-in-3.15.rst

```diff
@@ -1,57 +1,68 @@
 Pending Removal in Python 3.15
 ------------------------------
 
-* :class:`http.server.CGIHTTPRequestHandler` will be removed along with its
-  related ``--cgi`` flag to ``python -m http.server``.  It was obsolete and
-  rarely used.  No direct replacement exists.  *Anything* is better than CGI
-  to interface a web server with a request handler.
-
-* :class:`locale`: :func:`locale.getdefaultlocale` was deprecated in Python 3.11
-  and originally planned for removal in Python 3.13 (:gh:`90817`),
-  but removal has been postponed to Python 3.15.
-  Use :func:`locale.setlocale`, :func:`locale.getencoding` and
-  :func:`locale.getlocale` instead.
-  (Contributed by Hugo van Kemenade in :gh:`111187`.)
+* :mod:`ctypes`:
+
+  * The undocumented :func:`!ctypes.SetPointerType` function
+    has been deprecated since Python 3.13.
+
+* :mod:`http.server`:
+
+  * The obsolete and rarely used :class:`~http.server.CGIHTTPRequestHandler`
+    has been deprecated since Python 3.13.
+    No direct replacement exists.
+    *Anything* is better than CGI to interface
+    a web server with a request handler.
+
+  * The the :option:`!--cgi` flag to the :program:`python -m http.server`
+    command-line interface has been deprecated since Python 3.13.
+
+* :class:`locale`:
+
+  * The :func:`~locale.getdefaultlocale` function
+    has been deprecated since Python 3.11.
+    Its removal was originally planned for Python 3.13 (:gh:`90817`),
+    but has been postponed to Python 3.15.
+    Use :func:`~locale.getlocale`, :func:`~locale.setlocale`,
+    and :func:`~locale.getencoding` instead.
+    (Contributed by Hugo van Kemenade in :gh:`111187`.)
 
 * :mod:`pathlib`:
-  :meth:`pathlib.PurePath.is_reserved` is deprecated and scheduled for
-  removal in Python 3.15. Use :func:`os.path.isreserved` to detect reserved
-  paths on Windows.
+
+  * :meth:`.PurePath.is_reserved`
+    has been deprecated since Python 3.13.
+    Use :func:`os.path.isreserved` to detect reserved paths on Windows.
 
 * :mod:`platform`:
-  :func:`~platform.java_ver` is deprecated and will be removed in 3.15.
-  It was largely untested, had a confusing API,
-  and was only useful for Jython support.
-  (Contributed by Nikita Sobolev in :gh:`116349`.)
+
+  * :func:`~platform.java_ver` has been deprecated since Python 3.13.
+    This function is only useful for Jython support, has a confusing API,
+    and is largely untested.
```

Comment:
```suggestion
    and is largely untested.
    (Contributed by Nikita Sobolev in :gh:`116349`.)
```

Reply:
I wasn't going to do contributing lines in *pending removal*, because that's a reiteration of something that once appeared in *new deprecations*, so it feels duplicative. An exception though is when the timeline for a deprecation changes (such as your locale change).

Happy to reconsider though?